lib/active_admin/version.rb

```diff
@@ -1,3 +1,3 @@
 module ActiveAdmin
-  VERSION = '2.6.0'
+  VERSION = '2.6.1'
```

Comment:
We do this when we want to release a new patch level version of activeadmin. Don't worry about bumping this when you make PRs to gems, maintainers take care of this :+1:.